chartfx-math/src/main/codegen/de/gsi/math/ArrayMathGen.java

```diff
@@ -0,0 +1,570 @@
+package de.gsi.math;
+
+import de.gsi.dataset.utils.AssertUtils;
+
+/**
+ * Utility class containing static functions for working with arrays of
+ * different types.
+ *
+ * @author rstein
+ * @author Florian Enner
+ * @author Alexander Krimm
+ * @since 07 Jun 2020
+ */
+public class ArrayMathGen {
```

Comment:
Codacy found an issue: [The utility class name 'ArrayMathGen' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)